CONTRIBUTING.md

```diff
@@ -54,6 +54,26 @@ Signed-off-by: Your Name <YourName@example.org>
 in a single line, into the commit comment field. This can be automated by using a browser plugin like
 [DCO GitHub UI](https://github.com/scottrigby/dco-gh-ui).
 
+#### Steps to use the DCO browser plugin
+- Go to the plugin page on the [chrome web store](https://chrome.google.com/webstore/detail/dco-github-ui/onhgmjhnaeipfgacbglaphlmllkpoijo).
+- Alternatively, you could go to the [firefox addon page](https://addons.mozilla.org/en-US/firefox/addon/scott-rigby/) to add the extension to your browser.
+- Once you add the extension, right click on the extension in the toolbar of your browser and select `Options`. 
+- A dialog box will open up as shown below. Fill in your github name (not the handle) and email-id. 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/31214064/55411911-194c8500-5584-11e9-8b56-c8f94b6fa213.png" width="300">
+</p>
+
+- Then, whenever you perform a commit on github, the line `Signed-off-by: Your Name <Youremail>` will automatically appear in the commit description while making changes to a file as shown in the example below.
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/31214064/55412140-7ea07600-5584-11e9-8200-52cf068253ee.png" width="500">
```

Comment:
I suggest to use the markdown way to include images

    ![description](path to image)

This requires that the image is in the correct size before uploading it to github

Reply:
The problem with github markdown is that on top of the problem of not being able to resize images, you can't align an image. That's why I made the switch to html-like syntax.

Reply:
@GeorgLink, center aligned images will probably look better, but that's probably just me. After reading my reasons, let me know if you still want the markdown version, and I'll redo this part.

Reply:
I agree that the center alignment looks good. Does it still center align, if we load the image using markdown and wrap in in the `<p align="center">` environment?

Reply:
The image markdown isn't rendering that way

Reply:
Even if md is not perfect at aligning images, I would rather stick to md. Including HTML may cause trouble in the future, and adds some not needed complexity to a file that is supposed to be simple (with respect to mark down). I don't see it as a big trouble that the image appears in the middle, in the right, or in the left.

Reply:
Done